src/command_modules/azure-cli-storage/azure/cli/command_modules/storage/_params.py

```diff
@@ -335,7 +358,7 @@ def register_content_settings_argument(scope, settings_class):
 register_cli_argument('storage metrics update', 'api', help='Specify whether to include API in metrics. Applies to both hour and minute metrics if both are specified. Must be specified if hour or minute metrics are enabled and being updated.', choices=['enable', 'disable'])
 register_cli_argument('storage metrics update', 'retention', type=int, help='Number of days for which to retain metrics. 0 to disable. Applies to both hour and minute metrics if both are specified.')
 
-register_cli_argument('storage cors', 'max_age', type=int, help='The number of seconds the client/browser should cache a preflight response.')
+register_cli_argument('storage cors', 'max_age', type=int, help='The number of seconds the client/browser should cache a preflight response.', default="0")
```

Comment:
The help does say what it means. "The number of seconds..." So 0 means 0 seconds.

On the other point, yes it does NEED to be "0". Just 0 results in argparse not printing anything for the default. 

---
In reply to: [77392503](https://github.com/Azure/azure-cli/pull/804#discussion_r77392503) [](ancestors = 77392503)